index.html

```diff
@@ -47,64 +47,73 @@ <h5 class="header col s12 light">MSay Something Here...</h5>
       
        <!--   Mood Section   -->
       <div id="moodRow">
```

Comment:
Could you comment out the html so we use the javascript to display the elements. Your html will serve as the guide for the js.